src/components/Comment/index.tsx

```diff
@@ -189,11 +189,15 @@ const mapStateToProps = (
     }
   }
 
-  const info =
-    state.comments.byKey[createCommentKey({ fileName, line, versionId })];
+  const info = selectCommentInfo({
+    comments: state.comments,
+    fileName,
+    line,
+    versionId,
+  });
   return {
     initialComment: initialComment || null,
-    initialCommentText: info && info.pendingCommentText,
+    initialCommentText: info ? info.pendingCommentText : null,
```

Comment:
Sure, I'll add one. I can't think of a way to make it fail, though, since `undefined` is the expected value that the API should receive (as opposed to `null`). In any case, there's a test now.

Yeah, the old code was getting bitten by this TS quirk: https://github.com/microsoft/TypeScript/issues/13778